### PR TITLE
Update Makefile, add travis configuration

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Fixes https://github.com/giantswarm/builder/issues/65

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+notifications:
+    email: false
+
+before_install:
+- wget https://downloads.giantswarm.io/builder/0.11.0/builder
+- chmod +x ./builder
+- export PATH=$PATH:$PWD
+
+script:
+- make .gobuild
```
Comment (JosephSalisbury) [inline .travis.yml]: I'm doing this explicitly to make the travis output easier to read

**Comment by JosephSalisbury** [comment]
@zeisss || @xh3b4sd RFR

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- make ci-lint
```
Comment (zeisss) [inline .travis.yml]: afaik we said in the past we only want a `make ci` target and have everything in the makefile.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- make ci-test
+- make ci-build
+- ./builder --help
```
Comment (zeisss) [inline .travis.yml]: yes?

File: .travis.yml (hunk 2 above)
Comment (JosephSalisbury) [inline .travis.yml]: did we? i quite like having stuff split up, so travis can format it better

File: .travis.yml (hunk 3 above)
Comment (JosephSalisbury) [inline .travis.yml]: i like having proof that the tool _actually runs_ :D

Comment (zeisss) [inline .travis.yml]: then redirect the output to dev/null, add a comment and make it part of the pre_build steps?

File: .travis.yml (hunk 2 above)
Comment (zeisss) [inline .travis.yml]: I agree. So it would be the question of having a `ci` target that does the 3 makes in here. Not sure if we win sthg with that.

**Comment by zeisss** [comment]
:fishing_pole_and_fish:

**Comment by zeisss** [comment]
remove the builder bootstrapping foo that is useless, then LGTM

**Pull request merged**